ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_filter.proto

```diff
@@ -56,6 +56,13 @@ message InclusiveFilters {
   // the available packages at the time of the query.
   // Optional
   repeated InterfaceFilter interface_filters = 2;
+
+  // Whether to include a ``create_arguments_blob`` in the returned
+  // ``CreateEvent``.
+  // Use this to access the complete contract data in your API client
+  // for submitting it as a disclosed contract with future commands.
+  // Optional
+  bool include_create_arguments_blob = 3;
```

Comment:
I am wondering whether this the right place to specify whether we need a blob?
What does it mean that we want to have a blob for template_ids? Does it make sense at all? Also this will need to pair with merging rules as well, since there could be same interfaces per party... :thinking: 
AFAIR this flag was envisioned as part of the InterfaceFilter, so that we have fine-grained control over what is required per interface.

WDYT? @moritzkiefer-da

Reply:
Yeah that seems off to me. The original design specified that we put it in the interface filter. @mziolekda suggested in https://docs.google.com/document/d/1xpQI5AbEJdbpOWsjAUzOOTnJvTHTE4nDO1MnU7Ozng4/edit?disco=AAAAgEzXb0g that maybe we should also support it in templates (see my comments there for my thinking on that). But if we do support it for templates then I think we should allow specifying this for individual templates not once at the top-level of the filter.

Reply:
oh I see, looks like the discussion was in https://github.com/digital-asset/daml/pull/15012#discussion_r971900774. I'd still be inclined to go for the more fine-grained API here but in the end you own that API.

Reply:
After having further discussion with @nmarton-da and @mziolekda we have agreed to push it back to where it was. 

To summarise there are two arguments to have `include_create_arguments_blob` on the level of the InterfaceFilter:
1) there is no possible use case where  `create_arguments_blob` would be required without abstracting by the interface
2) to minimise the computation time of the blob as well as traffic amount in case there is no match of the template by interface.